DlgCreatePeer.cpp

```diff
@@ -192,9 +225,16 @@ QString DlgCreatePeer::create_dir(const QString &name){
 }
 
 void DlgCreatePeer::init_completed(system_call_wrapper_error_t res, QString dir, QString ram, QString cpu, QString disk){
-    ui->btn_create->setEnabled(true);
     if(res != SCWE_SUCCESS){
         CNotificationObserver::Instance()->Error("Coudn't create peer, sorry. Check if all software is installed correctly", DlgNotification::N_NO_ACTION);
+        ui->btn_create->setEnabled(true);
+        set_enabled_buttons(true);
+        ui->pb_peer->setValue(0);
+        ui->pb_peer->setEnabled(false);
+        ui->lbl_err_os->setText(tr("Failed to initialize environment"));
+        ui->lbl_err_name->setStyleSheet("QLabel {color : red}");
+        QDir dir(dir);
+        dir.removeRecursively();
         return;
     }
     CNotificationObserver::Instance()->Info("Initialization completed. Installing peer... Don't close terminal until instalation is compeleted", DlgNotification::N_NO_ACTION);
```

Comment:
"Initialization is completed. Installing peer... Don't close terminal until installation is finished." -- it would sound better

Reply:
@Erkin97